configs/example/gpufs/amd/AmdGPUOptions.py

```diff
@@ -247,16 +247,88 @@ def addAmdGPUOptions(parser):
         default="simple",
         help="register allocation policy (simple/dynamic)",
     )
+
+    parser.add_argument(
+        "--FunctionalTLB",
+        action="store_true",
+        help="Assumes TLB has no latency",
+    )
```

Comment:
Has functionalTLB been implemented in GPUFS??  I wasn't aware anyone was implementing this, but it would be another method to put the GPU model in a "fast-forwarding" mode.

Reply:
I removed this now. I am pushing a commit by someone else and this skipped my eye. Sorry about that!